dist/lib/checksum.js

```diff
@@ -1,4 +1,18 @@
 "use strict";
+/*
+ * Copyright The Notary Project Authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
Is this code block auto generated?

Reply:
@shizhMSFT This is actually expected. Because when we do `npm run build`, the compiler would just take the license headers as normal comments. For example, since we have the license header in the `src/lib/checksum.ts` as a comment block, then this comment block will be kept in the final `dist/lib/checksum.js`.
We could add `removeComments: true` into the tsconfig.json file. However, this will remove all the comments during compile.